src/general/GameWorld.cs

```diff
@@ -63,6 +63,7 @@ public GameWorld(WorldGenerationSettings settings, Species? startingSpecies = nu
         else
         {
             startingSpecies.BecomePlayerSpecies();
+            startingSpecies.Generation = 1;
```

Comment:
Could you add a comment here that the generation needs to be forced to be 1 in case it is different in the fossil file?